spec/policies/book_policy_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe BookPolicy, type: :policy do
+  let(:admin) { create(:user, access_level: 2)}
+  let(:user) { create(:user, access_level: 1) }
+
+  context "admin book creation" do
```

Comment:
Maybe group it by action?

describe "book creation" do
  context "when logged-in as admin" do
  context  "when logged-in as regular user"

Reply:
Got it! Thanks haha. It's my first time to write policy specs hahaha